packages/amplify-e2e-tests/sample.env

```diff
@@ -20,12 +21,9 @@ APPLE_KEY_ID=2QLZXKYJ8J
 APPLE_PRIVATE_KEY=----BEGIN PRIVATE KEY-----MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgIltgNsTgTfSzUadYiCS0VYtDDMFln/J8i1yJsSIw5g+gCgYIKoZIzj0DAQehRANCAASI8E0L/DhR/mIfTT07v3VwQu6q8I76lgn7kFhT0HvWoLuHKGQFcFkXXCgztgBrprzd419mUChAnKE6y89bWcNw----END PRIVATE KEY----
 APPLE_PRIVATE_KEY_2=----BEGIN PRIVATE KEY-----MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgIltgNsTgTfSzUadYiCS0VYtDDMFln/J8i1yJsSIw5g+gCgYIKoZIzj0DAQehRANCAASI8E0L/DhR/mIfTT07v3VwQu6q8I76lgn7kFhT0HvWoLuHKGQFcFkXXCgztgBrprzd419mUChAnKE6y89bWcNw----END PRIVATE KEY----
 
-#Used for delete test
-AWS_ACCESS_KEY_ID=<your-access-key>
-AWS_SECRET_ACCESS_KEY=<your-secret-access-key>
-CLI_REGION=<region>
-
 # Used for cleanup script
 CIRCLECI_TOKEN = '<your token>' # Token used for querying CircleCI to get the build details
 CIRCLE_PROJECT_USERNAME='<your github username>'
 CIRCLE_PROJECT_REPONAME='amplify-cli'
+
+AMPLIFY_PATH=
```

Comment:
Please add comment that explains what this does.

Something like `# A path to Amplify CLI executable. When working locally it should be set to `bin/amplify` file in the workspace, for example <insert example here> if running regular e2e test. If working locally with migration test it should point to older version of amplify. Absolute paths are recommended.`

Reply:
Done 295ec9040456514bd5e93edc16d9900b56be062a